docs/cloud-native-security/benchmark-rules.asciidoc

```diff
@@ -1,12 +1,47 @@
+The Benchmark Integrations page lets you view the cloud security posture (CSP) benchmark rules for the <<kspm, Kubernetes security posture management>> (KSPM) integration.
 
-To find the Benchmark Integrations page, go to **Manage -> CSP Benchmarks**. From there, to view the benchmark rules associated with an integration, select that integration's name.
+Benchmark rules are used by the Kubernetes Security Posture Management integration to identify configuration risks in Kubernetes. Benchmark rules are based on the Center for Internet Security's (CIS) https://www.cisecurity.org/cis-benchmarks/[secure configuration benchmarks for Kubernetes].
+
+Each benchmark rule checks to see if a specific Kubernetes resource is configured according to a CIS Benchmark. For example:
+
+* Ensure Kubernetes Secrets are encrypted using Customer Master Keys (CMKs) managed in AWS KMS
+* Ensure the default namespace is not in use
+
+When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Posture dashboard>>.
+
+To find the Benchmark Integrations page, go to **Manage -> CSP Benchmarks**. From there, you can view the benchmark rules associated with an existing integration by clicking its name, or create one if you haven't yet.
 
 [role="screenshot"]
 image::images/benchmark-rules.png[Benchmark rules page]
 
 You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
 
 NOTE: Benchmark rules are not editable.
+
+[discrete]
+=== How benchmark rules work
+
+. When the KSPM integration is deployed, and every four hours after that, {agent} fetches relevant Kubernetes resources.
+. Once a resource has been fetched, it is evaluated against all applicable benchmark rules.
+. Finding values of `pass` or `fail` indicate whether a resource passed a benchmark rule.
+
+
+[discrete]
+[[benchmark-rules-faq]]
+=== FAQ
+
+*Do benchmark rules cover multiple Kubernetes deployment types?*
+Yes. There are benchmark rule sets for self-managed and EKS-managed deployments.
+
+
+*How often are my resources evaluated with benchmark rules?*
+Resources are fetched and evaluated with benchmark rules every four hours.
+
+
+*Can I configure the KSPM integration's fetch cycle?*
+No, the fetch cycle is not configurable.
+
+*Can I contribute to the CSP ruleset?*
+You can't directly edit benchmark rules. The rules are defined https://github.com/elastic/csp-security-policies[in this repository], where you can raise issues with certain rules. They are written in https://www.openpolicyagent.org/docs/latest/policy-language/[Rego].
```

Comment:
Maybe we could add here an additional FAQ something like:

**How do I know what exact CIS benchmark version the ruleset relies on?**
Can be found in document's fields `rule.benchmark.name` and `rule.benchmark.version` in the datastreams `logs-cloud_security_posture.findings-default` and `logs-cloud_security_posture.findings_latest-default`

Reply:
This is great, thanks